core/thirdpartyapp/subgraph.py

```diff
@@ -0,0 +1,28 @@
+import requests
```

Comment:
please use request_helper,

you can check for example core/thirdpartyapp/farcaster.py.